Contentless/Program.cs

```diff
@@ -43,11 +44,62 @@ public static class Program {
         var excluded = config.ExcludedFiles.Select(Program.MakeFileRegex).ToArray();
         var overrides = Program.GetOverrides(config.Overrides).ToArray();
 
+        var referencesVersions = config.References.ToList().ToDictionary(x => x, x => (string)null, StringComparer.OrdinalIgnoreCase);
```

Comment:
Just call `ToDictionary` on the array here, `ToList` is unnecessary

Reply:
Fixed